woocommerce/woocommerce
# Add confetti to install success

### Changes proposed in this Pull Request:

We would like to add a confetti animation to the install success screen after a user activates a product.
The animation is from the existing `@automattic/components` package which we had to update to version 2.1.0.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch
2. Run `pnpm run build` (maybe `pnpm install` if checking out the branch doesn't trigger an update of packages)
3. Browse the [Marketplace for free extensions](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=extensions&path=%252Fextensions&category=free-extensions)
4. Click _Add to store_ on one product
5. Click _Install_
6. Click _Activate_ after the install finishes
7. See the confetti animation on activation success



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update @automattic/components package to 2.1.0.

#### Comment 

</details>


https://github.com/woocommerce/woocommerce/assets/1199991/d92acf4c-ef54-4fa9-95b0-95c29d5d8805
